src/components/Settings/preferences/SetupBanner.tsx

```diff
@@ -0,0 +1,28 @@
+import CampaignIcon from '@mui/icons-material/Campaign';
+import { Alert, Typography } from '@mui/material';
+
+interface SetupBannerProps {
+  button?: React.ReactNode;
+  content?: React.ReactNode;
+  title?: string;
+}
+
+export const SetupBanner: React.FC<SetupBannerProps> = ({
+  button,
+  content,
+  title,
+}) => {
+  return (
+    <Alert
+      severity="info"
+      iconMapping={{
+        info: <CampaignIcon fontSize="large" />,
+      }}
```

Comment:
This is a little simpler.

```suggestion
      icon={<CampaignIcon fontSize="large" />}
```